website/docs/commands/api-compatibility.md

```diff
@@ -171,53 +230,58 @@ Note that this list is subject to change as we continue to expand our API comman
+| <span id="memory">**MEMORY**</span> | DOCTOR | ➖ |  |
+|  | HELP | ➖ |  |
+|  | MALLOC-STATS | ➖ |  |
+|  | PURGE | ➖ |  |
+|  | STATS | ➖ |  |
+|  | [USAGE](server.md#memory-usage) | ➕ |  |
+| <span id="module">**MODULE**</span> | HELP | ➖ |  |
+|  | LIST | ➖ |  |
+|  | LOAD | ➖ |  |
+|  | LOADEX | ➖ |  |
+|  | UNLOAD | ➖ |  |
+| <span id="object">**OBJECT**</span> | ENCODING | ➖ |  |
+|  | FREQ | ➖ |  |
+|  | HELP | ➖ |  |
+|  | IDLETIME | ➖ |  |
+|  | REFCOUNT | ➖ |  |
 | <span id="pubsub">**PUB/SUB**</span> | [PSUBSCRIBE](analytics.md#psubscribe) | ➕ |  |
 |  | [PUBLISH](analytics.md#publish) | ➕ |  |
 |  | PUBSUB CHANNELS | ➖ |  |
+|  | PUBSUB HELP | ➖ |  |
 |  | PUBSUB NUMPAT | ➖ |  |
 |  | PUBSUB NUMSUB | ➖ |  |
+|  | PUBSUB SHARDCHANNELS | ➖ |  |
+|  | PUBSUB SHARDNUMSUB | ➖ |  |
 |  | [PUNSUBSCRIBE](analytics.md#punsubscribe) | ➕ |  |
 |  | [SUBSCRIBE](analytics.md#subscribe) | ➕ |  |
 |  | [UNSUBSCRIBE](analytics.md#unsubscribe) | ➕ |  |
-| <span id="scripting">**SCRIPTING**</span> | EVAL | ➕ |  |
+| <span id="scripting">**SCRIPTING**</span> | [EVAL](scripting-and-functions.md#eval) | ➕ |  |
 |  | EVAL_RO | ➖ |  |
-|  | EVALSHA | ➕ |  |
+|  | [EVALSHA](scripting-and-functions.md#evalsha) | ➕ |  |
 |  | EVALSHA_RO | ➖ |  |
 |  | SCRIPT DEBUG | ➖ |  |
-|  | SCRIPT EXISTS | ➕ |  |
-|  | SCRIPT FLUSH | ➕ |  |
+|  | [SCRIPT EXISTS](scripting-and-functions.md#script-exists) | ➕ |  |
+|  | [SCRIPT FLUSH](scripting-and-functions.md#script-flush) | ➕ |  |
+|  | SCRIPT HELP | ➖ |  |
 |  | SCRIPT KILL | ➖ |  |
-|  | SCRIPT LOAD | ➕ |  |
+|  | [SCRIPT LOAD](scripting-and-functions.md#script-load) | ➕ |  |
 | <span id="server">**SERVER**</span> | ACL | ➖ |  |
 |  | BGREWRITEAOF | ➖ |  |
 |  | [BGSAVE](checkpoint.md#bgsave) | ➕ |  |
-|  | [COMMAND](server.md#command) | ➕ |  |
-|  | [COMMAND COUNT](server.md#command-count) | ➕ |  |
-|  | COMMAND DOCS | ➖ |  |
-|  | COMMAND GETKEYS | ➖ |  |
-|  | COMMAND GETKEYSANDFLAGS | ➖ |  | 
-|  | [COMMAND INFO](server.md#command-info) | ➕ |  | 
-|  | COMMAND LIST | ➖ |  | 
 |  | [COMMITAOF](server.md#commitaof) | ➕ |  |
 |  | [CONFIG GET](server.md#config-get) | ➕ |  |
+|  | CONFIG HELP | ➖ |  |
+|  | CONFIG RESETSTAT | ➖ |  |
+|  | CONFIG REWRITE | ➖ |  |
 |  | [CONFIG SET](server.md#config-set) | ➕ |  |
 |  | [DBSIZE](server.md#dbsize) | ➕ |  |
+|  | DEBUG | ➖ | Internal command |
 |  | [FLUSHALL](server.md#flushall) | ➕ |  |
 |  | [FLUSHDB](server.md#flushdb) | ➕ |  |
 |  | [LASTSAVE](checkpoint.md#lastsave) | ➕ |  |
-|  | LATENCY DOCTOR | ➖ |  |
-|  | LATENCY GRAPH | ➖ |  |
-|  | [LATENCY HISTOGRAM](server.md#latency-histogram) | ➕ |  |
-|  | LATENCY HISTORY | ➖ |  |
-|  | LATENCY LATEST | ➖ |  |
-|  | [LATENCY RESET](server.md#latency-reset) | ➕ |  |
-|  | LOLWUT | ➕ |  |
-|  | MEMORY DOCTOR | ➖ |  |
-|  | MEMORY MALLOC-STATS | ➖ |  |
-|  | MEMORY PURGE | ➖ |  |
-|  | MEMORY STATS | ➖ |  |
-|  | [MEMORY USAGE](server.md#memory-usage) | ➕ |  |
-|  | MODULE | ➖ |  |
+|  | [LOLWUT](server.md#lolwut) | ➕ |  |
```

Comment:
i don't think Garnet has support for the LOLWUT command. Need to remove here and the description section for it.